File: app/src/main/java/workshop/akbolatss/xchangesrates/base/LoadingView.java
package workshop.akbolatss.xchangesrates.base;

/**
 * Author: Akbolat Sadvakassov
 * Date: 04.01.2018
 */

public interface LoadingView {

    void onShowLoading();

    void onHideLoading();

    void onNoContent(boolean isEmpty);
}

File: app/src/main/java/workshop/akbolatss/xchangesrates/base/LoadingView.kt
package workshop.akbolatss.xchangesrates.base

/**
 * Author: Akbolat Sadvakassov
 * Date: 04.01.2018
 */

interface LoadingView {

    fun onShowLoading()

    fun onHideLoading()

    fun onNoContent(isEmpty: Boolean)
}
